manager/controllers/app/blueprint_controller.go

```diff
@@ -154,35 +158,72 @@ func (r *BlueprintReconciler) applyChartResource(log logr.Logger, ref string, va
 	if username != "" && password != "" {
 		err := r.Helmer.RegistryLogin(hostname, username, password, insecure)
 		if err != nil {
-			return ctrl.Result{}, errors.WithMessage(err, ref+": failed chart pull")
+			return ctrl.Result{}, errors.WithMessage(err, chartSpec.Name+": failed chart pull")
 		}
 	}
 
-	err := r.Helmer.ChartPull(ref)
+	err := r.Helmer.ChartPull(chartSpec.Name)
 	if err != nil {
-		return ctrl.Result{}, errors.WithMessage(err, ref+": failed chart pull")
+		return ctrl.Result{}, errors.WithMessage(err, chartSpec.Name+": failed chart pull")
 	}
-	chart, err := r.Helmer.ChartLoad(ref)
+	chart, err := r.Helmer.ChartLoad(chartSpec.Name)
 	if err != nil {
-		return ctrl.Result{}, errors.WithMessage(err, ref+": failed chart load")
+		return ctrl.Result{}, errors.WithMessage(err, chartSpec.Name+": failed chart load")
 	}
 
+	kubeNamespace := args["metadata"].(map[string]interface{})["namespace"].(string)
```

Comment:
I don't think that we pass metadata as args anymore. Does this revert https://github.com/IBM/the-mesh-for-data/pull/155 ?

Reply:
removed